src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/ServiceProviderComment.java

```diff
@@ -0,0 +1,59 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split;
+
+/**
+ * Example FTX+SPC+++red blood cell seen, Note low platelets'
+ */
+@RequiredArgsConstructor
+@Getter
+@EqualsAndHashCode(callSuper = false)
+public class ServiceProviderComment extends Segment {
+    private static final String KEY = "FTX";
+    private static final String QUALIFIER = "SPC";
+    private static final String KEY_QUALIFIER = KEY + PLUS_SEPARATOR + QUALIFIER;
+    private static final int FOUR = 4;
+
+    private final String serviceProviderComment;
+
+    public static ServiceProviderComment fromString(String edifactString) {
+        if (!edifactString.startsWith(ServiceProviderComment.KEY_QUALIFIER)) {
+            throw new IllegalArgumentException("Can't create " + ServiceProviderComment.class.getSimpleName() + " from " + edifactString);
+        }
+        String[] split = Split.byPlus(
+            Split.bySegmentTerminator(edifactString)[0]
+        );
+        return new ServiceProviderComment(split[FOUR]);
+    }
+
+    @Override
+    public String getKey() {
+        return KEY;
+    }
+
+    @Override
+    public String getValue() {
+        return String.join(PLUS_SEPARATOR,
+            QUALIFIER,
+            StringUtils.EMPTY,
```

Comment:
I'm curious about why we're using `StringUtils.EMPTY` here?

Reply:
This is needed to basically have the "+++" for the value. If you would look at the EDIFACT message example, it would look like this: FTX+SPC+++red blood cell seen, Note low platelets'